tests/test_routes.py

```diff
@@ -130,6 +130,20 @@ def test_get_wishlist(self):
         self.assertEqual(data['wishlist_name'], wishlist_name)
         self.assertEqual(data['created_date'], str(created_date))
 
+    def test_delete_wishlist(self):
+        """It should Delete a Wishlist"""
+        # get the id of a wishlist
+        wishlist = self._create_wishlists(1)[0]
+        resp = self.client.delete(f"{BASE_URL}/{wishlist.id}")
+        self.assertEqual(resp.status_code, status.HTTP_204_NO_CONTENT)
+
+    def test_delete_sad_path_wishlist(self):
+        """It should throw an error code"""
+        # get the id of a wishlist
+        wishlist = self._create_wishlists(1)[0]
+        resp = self.client.delete(f"{BASE_URL}/{wishlist.id+100}")
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
```

Comment:
the hard coded "+100" might be a concern. Maybe assert that it definately does not exist prior to deleting it.